### ISSUE TITLE
🐛 [Bug] Savannah Huntress movement / stand state

### ISSUE THREAD

**Issue by ghost** [vmangos/core#2759]
## 🐛 Bug report
There are 4 Savannah Huntresses which have UNIT_STAND_STATE_SLEEP and at the same time random movment, which looks strange:
https://github.com/user-attachments/assets/d1069221-57a8-4035-9344-3fe4a2bf5719

#2725 was obviously the wrong way to fix this, so I closed the PR.

### Expected behavior
They should either sleep or walk around, but not both at the same time.

### Steps to reproduce
```
.go cr 14119
.go cr 14117
.go cr 14116
.go cr 14108
```

### Version & Environment
Client Version: 1.12.1.5875
Commit Hash: 8313afebd39ffa51eadebf79bbfee6874b2252f5

OS Client: Ubuntu 22.04 (Wine)
OS Server: Ubuntu 22.04

**Comment by mserajnik** [comment]
> There are 4 Savannah Huntresses which have UNIT_STAND_STATE_SLEEP and at the same time random movment, which looks strange:
> https://github.com/user-attachments/assets/d1069221-57a8-4035-9344-3fe4a2bf5719

Don't see what's wrong here, behaves just like my cat. 😀

**Comment by FlagFlayer** [comment]
I checked on the Classic PTR and I believe the implementation of UNIT_STAND_STATE_SLEEP is what's incorrect.

In Classic when the 4 creatures you've mentioned spawn they will be walking around (random movement) and at a random point will go into this "sleep state" in which for the next 2-5 minutes (random amount of time) the creature will sit down, afterwhich the creature will again stand up and walk around until the next random time it decides to go back into the sleep state

If someone wants to go stare at cats for hours to figure out the mechanics of this lovely thing..... Be my guest....

**Comment by codestothestars** [comment]
All this needs is a sniff and it will be easy to figure out from the data. Park a character there for a half hour and go do something else, no staring required.

**Comment by FlagFlayer** [comment]
According to brotalnia we can't parse the sniffs :/

**Comment by codestothestars** [comment]
We can parse sniffs from the live client build (not PTR) once support is added to the packet parser.

**Comment by ratkosrb** [comment]
Nobody adds support for new ClassicEra builds.

**Comment by ghost** [comment]
> In Classic when the 4 creatures you've mentioned spawn they will be walking around (random movement) and at a random point will go into this "sleep state" in which for the next 2-5 minutes (random amount of time) the creature will sit down, afterwhich the creature will again stand up and walk around until the next random time it decides to go back into the sleep state

Is this the same for all Savannah Huntresses?

**Comment by Daribon** [comment]
> We can parse sniffs from the live client build (not PTR) once support is added to the packet parser.

I will probably add both live and PTR builds to TC parser it in the future, since it already has partial support. Just need the build numbers added.

So don't let parser prevent you from sniffing on PTR.

**Comment by FlagFlayer** [comment]
> Is this the same for all Savannah Huntresses?

No, only specific ones, like the ones mentioned above, although at least one is missing from the list (I saw 5 creatures behave this way in that area whereas on vmangos there's only the 4 mentioned)

**Comment by ghost** [comment]
There are also 2 Savannah Prowlers with the same issue: 14175, 14187
